zend/classimpl.cpp

```diff
@@ -1354,7 +1354,9 @@ zend_class_entry *ClassImpl::initialize(ClassBase *base, const std::string &pref
     {
         // install iterator functions
         entry.get_iterator = &ClassImpl::getIterator;
+#if PHP_VERSION_ID < 70300
         entry.iterator_funcs.funcs = IteratorImpl::functions();
```

Comment:
I wasn't sure if it was no longer required as the details in the php README.internals wasn't very clear to me.

Reply:
I think they are still required, but they have to be allocated now, as there is a iterator_funcs_ptr in place of this function. However, that iterator_funcs_ptr does not contain `funcs`, so something else has changed there.